.github/workflows/linting.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Make sure each commit in the PR is within reviewable size
         uses: ./gh-actions/commit-size
         with:
-          size: 200
+          size: 250
```

Comment:
If this gets updated for any reason, could you mention this change in the corresponding commit message?